drupalcat/config/install/core.entity_form_display.node.page.default.yml

```diff
@@ -0,0 +1,60 @@
+uuid: 54b7a11a-a7a8-4d11-b900-b0c63282aba5
```

Comment:
los uuids ligan la configuración a un sitio concreto, no los necesitamos
http://enzolutions.com/articles/2014/08/27/understanding-configuration-management-in-drupal-8/